lua/class_definitions.lua

```diff
@@ -84,11 +84,6 @@ classes = {
             nv_cached = { type = "bool", writable = true },
         },
         functions = {
-            { name = "add_effect", rval = nil, args = { "string", "int" } },
```

Comment:
If I'm not mistaken, StatsFromSkills mod uses these calls to apply stat changes without modifying the base stat values.